content/applications/inventory_and_mrp/barcode/setup/hardware.rst

```diff
@@ -4,64 +4,78 @@ Set up your barcode scanner
-
-Find the barcode scanner that suits your needs
-==============================================
-
-The 3 recommended type of barcode scanners to work with the Odoo
-**Inventory** and **Barcode Scanning** apps are the **USB scanner**, **the bluetooth
-scanner** and the **mobile computer scanner**.
-
-.. image:: hardware/hardware02.png
-    :align: center
-
--   If you scan products at a computer location, the **USB scanner** is the
-    way to go. Simply plug it in the computer to start scanning. Just
-    make sure when you buy it that the scanner is compatible with
-    your keyboard layout or can be configured to be so.
-
--   The **bluetooth scanner** can be paired with a smartphone or a tablet and
-    is a good choice if you want to be mobile but don't need a big
-    investment. An approach is to log in Odoo on you smartphone, pair
-    the bluetooth scanner with the smartphone and work in the
-    warehouse with the possibility to check your smartphone
-    from time to time and use the software 'manually'.
-
--   For heavy use, the **mobile computer scanner** is the handiest solution.
-    It consists of a small computer with a built-in barcode scanner.
-    This one can turn out to be a very productive solution, however
-    you need to make sure that is is capable of running Odoo smoothly.
-    The most recent models using Android + Google Chrome or Windows +
-    Internet Explorer Mobile should do the job. However, due to the
-    variety of models and configurations on the market, it is
-    essential to test it first.
-
-Configure your barcode scanner
-==============================
+Follow this guide to choose and set up a barcode scanner compatible with Odoo's *Inventory* and
+*Barcode* apps.
+
+.. image:: hardware/barcode-scanner.png
+   :align: center
+   :alt: An image of an example barcode scanner.
+
+Scanner types
+=============
+
+Before setting up a barcode scanner, it is important to determine which scanner type best meets the
+needs of the business. There are three main types, each with their own benefits and use cases:
+
+- **USB scanners** are connected to a computer, and are suitable for businesses that scan products
+  at a fixed location, like at the checkout in a grocery store. Ensure the chosen USB scanner is
+  compatible with the keyboard layout of the computer.
+
+- **Bluetooth scanners** pair with a smartphone or tablet, making them an ideal cost-effective and
+  portable barcode scanner option. In this scenario, Odoo is installed on the smartphone, allowing
+  warehouse operators to handle operations, and check stock directly through their mobile devices.
+
+- **Mobile computer scanners** are mobile devices with a built-in barcode scanner. First, ensure the
+  device can run the Odoo mobile app properly. Recent models that use Android OS with the Google
+  Chrome browser, or Windows OS with Microsoft Edge, should work. However, testing is crucial due to
+  the variety of available models and configurations.
+
```

Comment:
Suggestion to point to Inventory hardware page on the main site, which lists specific scanner models

```suggestion

.. seealso::
   `Compatible hardware with Odoo Inventory <https://www.odoo.com/app/inventory-hardware>`_

```